schema-engine/connectors/sql-schema-connector/src/warnings/generators.rs

```diff
@@ -61,6 +61,8 @@ pub(crate) struct Warnings {
     pub(crate) row_level_ttl: Vec<Model>,
     /// Warn about non-default unique deferring setup
     pub(crate) non_default_deferring: Vec<ModelAndConstraint>,
+    /// Warn about comments
+    pub(crate) commented_objects: Vec<Object>,
```

Comment:
Nitpick? It currently reads as objects that have been commented out
```suggestion
    pub(crate) objects_with_comments: Vec<Object>,
```